### PR TITLE
revert mentions for chat preview

### PR DESCRIPTION
**Pull request by flexsurfer**
revert mentions for chat preview

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status_go.html))?
- [ ] Have you tested changes with mobile?
- [ ] Have you tested changes with desktop?

File: protocol/messenger_chats.go [hunk 1]
```diff
@@ -51,11 +53,15 @@ func (m *Messenger) ChatsPreview() []*ChatPreview {
...
 				chatPreview.ContentType = chat.LastMessage.ContentType
 				if chat.LastMessage.ContentType == protobuf.ChatMessage_TEXT_PLAIN {
-					if len(chat.LastMessage.Text) > 200 {
-						chatPreview.Text = chat.LastMessage.Text[:200]
+					if len(chat.LastMessage.Text) > 100 {
+						chatPreview.Text = chat.LastMessage.Text[:100]
 					} else {
 						chatPreview.Text = chat.LastMessage.Text
 					}
+					if strings.Contains(chatPreview.Text, "@0x") {
```
Comment (cammellos) [inline protocol/messenger_chats.go]: I think there's a field already, `Mentions` that you should be able to check, something like `len(chatPreview.Mentions)>0`

Comment (cammellos) [reply protocol/messenger_chats.go]: on `LastMessage`

Comment (flexsurfer) [reply protocol/messenger_chats.go]: i mean if there is no visible mention we don't need to send it, we want to send only if it's in the first 100 chars

Comment (cammellos) [reply protocol/messenger_chats.go]: Oh I see, you are right, I missed that. You might still want to do it performance wise though, as 
`len(chatPreview.LastMessage.Mentions) > 0 && strings.Contains(chatPreview.Text, "@0x")` should be faster in most cases. Though not sure if it's a necessary optimization.

Comment (cammellos) [reply protocol/messenger_chats.go]: What about markdown btw?

Comment (flexsurfer) [reply protocol/messenger_chats.go]: this PR is only for mentions

Comment (flexsurfer) [reply protocol/messenger_chats.go]: markdown is not so critical in the preview imho

Comment (flexsurfer) [reply protocol/messenger_chats.go]: added GetSimplifiedText for markdown, thanks

**Comment by flexsurfer** [comment]
ping @cammellos

**Review by cammellos — APPROVED** [review]

**Pull request merged**